deploy/olm-catalog/knative-eventing-operator/0.11.0/eventing_v1alpha1_knativeeventing_crd.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: eventings.operator.knative.dev
+spec:
+  group: operator.knative.dev
+  names:
+    kind: Eventing
+    listKind: EventingList
+    plural: eventings
+    singular: eventing
+    shortNames:
+      - ke
```

Comment:
do we really need this shortname ? /cc @cardil

Reply:
hmm, I started this PR before the shortname discussion. What's the latest situation there ? :rofl:

Reply:
Short names should be removed

Reply:
hmm, hold on :)

this OLM metadata is for the 0.11 version of the operator. In that version, short name is still there: https://github.com/knative/eventing-operator/blob/v0.11.0/config/300-eventing-v1alpha1-knativeeventing-crd.yaml#L12

So, let's keep this one for consistency. The CRD in this PR is identical to https://github.com/knative/eventing-operator/blob/v0.11.0/config/300-eventing-v1alpha1-knativeeventing-crd.yaml#L12